Readme.md

```diff
@@ -40,6 +40,7 @@ The `options` argument allows you to customize the client with the following pro
 - endpoint: to override the SOAP service's host specified in the `.wsdl` file.
 - request: to override the [request](https://github.com/request/request) module.
 - httpClient: to provide your own http client that implements `request(rurl, data, callback, exheaders, exoptions)`.
+- forceSoap12: to set proper headers for SOAP v1.2
```

Comment:
alternatively, is there not a way to infer the soap version from the WSDL itself?  Seems like `node-soap` should set the proper headers for the soap version.